.gitignore

```diff
@@ -12,3 +12,4 @@ dist
 /package-lock.json
 /yarn.lock
 .npmrc
+/my-app
```

Comment:
Please remove this change which is only related to your local setup.

Reply:
Done